Framework/Algorithms/src/Rebin.cpp

```diff
@@ -135,15 +136,25 @@ std::map<std::string, std::string> Rebin::validateInputs() {
   // validate the rebin params, and outside default mode, reset them
   MatrixWorkspace_sptr inputWS = getProperty(PropertyNames::INPUT_WKSP);
   std::vector<double> rbParams = getProperty(PropertyNames::PARAMS);
-  try {
-    std::vector<double> validParams = rebinParamsFromInput(rbParams, *inputWS, g_log, binMode);
-    // if the binmode has been set, force the rebin params to be consistent
-    if (binMode != BinningMode::DEFAULT) {
-      setProperty(PropertyNames::PARAMS, validParams);
-      rbParams = validParams;
+  if (inputWS == nullptr) {
+    // The workspace could exist, but not be a MatrixWorkspace, e.g. it might be
+    // a group workspace. In that case we don't want a validation error for the
+    // Rebin parameters.
+    std::string inputWsName = getProperty(PropertyNames::INPUT_WKSP);
```

Comment:
We don't seem to change this. Could it be const &?
```suggestion
    const std::string &inputWsName = getProperty(PropertyNames::INPUT_WKSP);
```